ui/tests/acceptance/volumes-list-test.js

```diff
@@ -69,15 +69,17 @@ module('Acceptance | volumes list', function(hooks) {
 
     const volumeRow = VolumesList.volumes.objectAt(0);
 
-    const controllerHealthStr = volume.controllersHealthy > 0 ? 'Healthy' : 'Unhealthy';
+    const controllerHealthStr = volume.controllersHealthy > 0 ?
+      `Healthy (${volume.controllersHealthy}/${volume.controllersExpected})` :
+      volume.controllerRequired ?
+        'Unhealthy (${volume.controllersHealthy}/${volume.controllersExpected})' :
+        'Node Only';
```

Comment:
This isn't enforced by eslint, but we're typically anti-nested ternary. I'd probably write this as

```js
let controllerHealthStr = 'Node Only';
if (volume.controllerRequired) {
  const healthy = volume.controllersHealthy;
  const expected = volume.controllersExpected;
  const isHealthy = healthy > 0;
  controllerHealthStr = `${isHealthy ? 'Healthy' : 'Unhealthy'} (${healthy}/${expected})`;
}
```